frontend/common/src/helpers/localize.test.ts

```diff
@@ -44,5 +49,16 @@ describe("localize helper tests", () => {
       expect(localizePath("admin/users", "en")).toBe("en/admin/users");
       expect(localizePath("fr/admin/users", "en")).toBe("en/admin/users");
     });
+    test("works on single slash", () => {
+      expect(localizePath("/", "en")).toBe("/en/");
+    });
+    // test("works on empty string", () => {
+    //   expect(localizePath("", "en"))).toBe("en/");
+    // });
```

Comment:
I think this was just my mistake. The other expected strings don't end with a slash
```suggestion
     test("works on empty string", () => {
       expect(localizePath("", "en"))).toBe("en");
     });
```

Reply:
I've uncommented the test with the corrected expected value.  All tests are green now.